lms/templates/student_account/login.underscore

```diff
@@ -1,14 +1,24 @@
 <div class="js-form-feedback" aria-live="assertive" tabindex="-1">
 </div>
 
-<% if ( context.createAccountOption !== false && !context.syncLearnerProfileData) { %>
+<% if ( context.createAccountOption !== false && !context.syncLearnerProfileData && context.enterpriseName == "") { %>
 <div class="toggle-form">
     <span class="text"><%- gettext("First time here?") %></span>
     <a href="#login" class="form-toggle" data-type="register"><%- gettext("Create an Account.") %></a>
 </div>
 <% } %>
 
-<h2><%- gettext("Sign In") %></h2>
+<% if (context.enterpriseName) { %>
+    <% if (context.pipelineUserDetails.email) { %>
+        <h2><%- gettext("Sign in to continue learning as {email}").replace("{email}", context.pipelineUserDetails.email) %></h2>
+    <% } else { %>
+        <h2><%- gettext("Sign in to continue learning") %></h2>
+    <% } %>
+    <p><%- gettext("You already have an edX account with your {enterprise_name} email address. Going forward, your account information will be updated and maintained by {enterprise_name}. You can view your information or unlink from {enterprise_name} anytime in your Account Settings.").replace(/{enterprise_name}/g, context.enterpriseName) %> </p>
```

Comment:
This is only true if `sync_learner_profile_data` is set to `True` on the IdP config.

Reply:
Can you elaborate it more? Do u mean that we need to add a check of syncLearnerProfileData?
if (context.syncLearnerProfileData) {
      <p><%- gettext("You already have an edX account ..... 
}

Reply:
@georgebabey can u confirm this more? These details/messages will be displayed when we have an enterprise on that screen. Do we need to another condition as per @douglashall ? ( I pointed above)

Reply:
Yes I believe Doug is saying we should wrap the message `Going forward, your account information will be updated and maintained by {enterprise_name}.` with the if check for `sync_learner_profile_data`